### PR TITLE
cylc 8 gantt chart: more on Rose

### PR DESCRIPTION
**Pull request by matthewrmshin**
Time estimate for Python 3 and packaging activities.
Add Rose Config-Edit.
Also fix typo *chartt* -> *chart*.

### CODE REVIEW

**Comment by matthewrmshin** [comment]
@hjoliver please review. (For some reason, I still can't assign reviewer, etc.)

**Comment by hjoliver** [comment]
> _chartt_ -> _chart_.

Aww, that was deliberate, following Gan(tt)!

File: docs/cylc-8-tasks.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
 ### Major Updates and Changes
  - Initially compiled after the December 2018 Development Workshop.
- - Updated 25 Feb 2019: added Gantt Chartt.
+ - Updated 25 Feb 2019: added Gantt Chart.
 
 ### New Terminology
 _suite daemon_  (original) -> _suite server program_ (more recent) -> __workflow service__ (WS) (new)
 
-## Projectt Gantt Chartt
+## Projectt Gantt Chart
```
Comment (hjoliver) [inline docs/cylc-8-tasks.md]: But you missed Projec(tt) 😁

Comment (matthewrmshin) [reply docs/cylc-8-tasks.md]: Haha. I'll correct my name next.

**Pull request merged**